source/main/modules/export/write-defaults.ts

```diff
@@ -26,14 +29,18 @@ export default async function writeDefaults (
     title: path.basename(sourceFile, path.extname(sourceFile))
   }
 
-  // Use an HTML template if applicable
+  // Pick the appropraite template, if applicable
   if (writer === 'html') {
     let tpl = await fs.readFile(path.join(__dirname, 'assets/export.tpl.htm'), { encoding: 'utf8' })
```

Comment:
What I wanna do with this code is write the templates to a custom resource dir for Zettlr (as I'm doing with the defaults files) and write this template into the HTML defaults file by default, so it can be easily adapted by the users. It would be cool to do this, and use the assets-provider for this! Gives us the ability to both use templates AND reference-docs without writing additional code!